### PR TITLE
updates for v5 beta

### PR DESCRIPTION
**Pull request by AlexMarchini**

### CODE REVIEW

File: utils/objectUtils.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const entityType = { SUITE: 'SUITE', TEST: 'STEP' },
     getStartLaunchObject = (options = {}) => ({
         launch: process.env.RP_LAUNCH_NAME || options.launchname || 'Unit Tests',
         tags: options.tags,
-        start_time: new Date().valueOf()
+        startTime: new Date().valueOf()
```
Comment (BorisOsipov) [inline utils/objectUtils.js]: It is strange. Looks like in RP v5 reporting compatibility with v4 is broken. I believed RP team has fixed this.@DzmitryHumianiuk ?
Also this changes have to be in rp-js client to fix other integration.

Comment (DzmitryHumianiuk) [reply utils/objectUtils.js]: RPv5 api will accept both `start_time` and `startime` attribute.
as a backward compatibility.

@pbortnik and @ihar-kahadouski can share details.

Comment (BorisOsipov) [reply utils/objectUtils.js]: @DzmitryHumianiuk yes it should. But seems it doesn't work if it is required to make such changes in reporter

**Comment by AmsterGet** [comment]
Closed as irrelevant.
The new package `@reportportal/agent-js-jest` with versions 5.* is only compatible with RP version 5.*.

**Pull request closed without merging**